modules/cutadapt/1.0/config/default.yaml

```diff
@@ -0,0 +1,43 @@
+lcr-modules:
+
+    cutadapt:
+
+        inputs:
+            # Available wildcards: {seq_type} {sample_id}
+            sample_fastq_1: "__UPDATE__"
+            sample_fastq_2: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            fastqc: ""
+            cutadapt: ""
+            # Provide adapter sequences to trim below
+            # These default options are Illumina universal adapters
+            # This will be passed with -a option of cutadapt
+            forward_a: "AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC"
+            # This will be passed with -A option of cutadapt
+            reverse_a: "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT"
+
+        conda_envs:
+            cutadapt: "{MODSDIR}/envs/cutadapt-4.9.yaml"
+
+        threads:
+            fastqc: 4
+            cutadapt: 4
+
+        resources:
+            fastqc:
+                mem_mb: 2000
```

Comment:
I think from what I used with the samples in hand that this is more dictated by the number of cores rather than memory. I don't know for sure though and I don't have WGS fastqs on hand to test. The files I used to run with this module are more than the 2Gb specified here but run with no issues. This low memory mb also lets them to get resources allocated very quickly in the numbers queue.

Reply:
perfect!